CHANGES.rst

```diff
@@ -1,8 +1,6 @@
 Version 0.2.0
 =============
 
-Release TBD
```

Comment:
We usually put the date of the release in the change logs. See `Henson`: https://github.com/iheartradio/Henson/blob/master/CHANGES.rst